src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -214,23 +192,38 @@ public boolean isDependentArgumentAllowed(final Class<?> dependentClass) {
             if (rf != null) {
                 filters.add(rf);
             } else if (disableToolDefaultReadFilters) {
-                // TODO: this behaviour is modified wrt to GATK in https://github.com/broadinstitute/gatk/pull/2365
                 // include in the read instances since the tool one is disabled, but it is provided by the user
                 filters.add(toolDefaultReadFilters.get(s));
             }
         });
         return filters;
     }
 
+    /**
+     * Get the list of default plugins used for this instance of this descriptor. Used for help/doc generation.
+     * @return List of T
+     */
+    @Override
+    public List<Object> getDefaultInstances() { return new ArrayList<>(toolDefaultReadFilters.values()); }
```

Comment:
And we need a test for this in our trimming tool, to check that we are really disabling everything...